### DISCUSSION TITLE
Is primevue blocks updated?

### DISCUSSION THREAD

**Discussion by akorajac** [primefaces/community#2269] [PrimeVue Blocks]
I purchased a license of primevue blocks. They dont seem to work on primevue V4 (especially the sidebar layouts).

**Comment by Coffee2CodeNL** [comment]
No

**Comment by akorajac** [comment]
it's been 10 days, no answer from the support. Paid for a product that doesn't seem to work and no support for it. Great 👍

**Reply by Coffee2CodeNL** [reply]
I feel the same, no refunds but also no gestures from the devs for the misbuy.

Try discord.

**Comment by hypervision-am** [comment]
Can you please, when will it be updated?
